proto/ls_state.proto

```diff
@@ -30,3 +30,11 @@ message LinRegUniLSState {
   Vector regression_coeffs = 1; // regression coefficients
   double var = 2; // variance of the noise
 }
+
+message MFAState {
+  Vector mu = 1;
+  Vector psi = 2;
+  Matrix eta = 3;
+  Matrix lambda = 4;
+  Matrix psi_inverse = 5;
```

Comment:
do you need a matrix here? it will be diagonal as well right?

Reply:
Yes it is diagonal since psi is just a vector that stores the diagonal of Psi in Lambda*Lambda^T + Psi, if we don't pass it through the interface we can just remove it from proto? Should we do the same for parameters mutilde and beta once we have automatic initialization?

Reply:
Fine for me removing psi_inverse from here. I would still keep the parameters mutilde and beta, to test different initialisations strategies